app/posts/find-teacher-training/2021-08-16-standardised-entry-requirement-descriptions.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Standardising entry requirement content
+description: Why we have made the descriptions of entry requirements more consistent
+date: 2021-08-16
+---
+
+For the 2021 to 2022 cycle, providers were asked to describe the academic qualification entry requirements for each course as a single free text field.
+
+As there is a minimum initial teacher training entry requirement across all courses, this led to the standard entry requirements being described in different ways by different providers.
+
+For instance:
+
+> GCSE Maths, English and Science at Grade C or higher or equivalent
+
+> GCSEs: At least a Grade C or Grade 4 at GCSE (or equivalent) in English Language, Mathematics and Science is essential for all applicants.
+
+> GCSE in English (or approved equivalent) at grade C/Level 4 or above GCSE in Mathematics (or approved equivalent) at grade C/Level 4 or above GCSE in Science (or approved equivalent) at grade C/Level 4 or above
+
+These minor inconsistencies may be confusing for candidates, and the requirements weren’t always written in the easiest to understand way.
+
+Instead, for the 2022 to 2023 cycle, we have changed the Publish interface to only ask providers to specify the entry requirements that may differ across courses, namely the [acceptance of pending GCSEs and GCSE equivalency tests](/publish-teacher-training-courses/pending-gcses-equivalency-tests/) and the [minimum degree class required](/publish-teacher-training-courses/degree-entry-requirements/).
+
+These changes allow us use consistent language to describe the standard entry requirements.
+
+## New content structure
+
+### Primary courses
+
+The standardised content is:
+
+> An undergraduate degree, or equivalent.
+>
+> GCSE grade 4 (&#8203;C) or above in English, maths and science, or equivalent qualification.
+
+Then the following paragraph is constructed from the provider’s answers to the pending GCSEs and equivalency tests questions, for example:
+
+> We’ll consider candidates who are currently taking GCSEs. We will not consider candidates who need to take GCSE equivalency tests.
+
+or
+
+> We’ll consider candidates who are currently taking GCSEs. We’ll consider candidates who need to take a GCSE equivalency test in English, maths, or science.
+
+Where providers do accept equivalency tests, they are asked to give details, and this free text field is shown at the end.
+
+### Secondary courses
+
+For Secondary courses, a standardised paragraph is added about the degree subject:
```

Comment:
Should ‘primary’ and ‘secondary’ be capitalised? 

```suggestion
For secondary courses, a standardised paragraph is added about the degree subject:
```

Reply:
🤷🏻‍♂️